tests/admin.test.ts

```diff
@@ -117,3 +118,83 @@ describe('retroactive attendance submission', () => {
     expect(staffActivityResponse.activity[1].type).toEqual(ActivityType.ATTEND_EVENT_AS_STAFF);
   });
 });
+
+describe('email retrieval', () => {
+  test('gets all the emails of stored users', async () => {
+    const conn = await DatabaseConnection.get();
+    const users = UserFactory.create(5);
+    const emails = users.map((user) => user.email.toLowerCase());
+    const [proxyUser] = UserFactory.with({ accessType: UserAccessType.ADMIN });
+
+    await new PortalState()
+      .createUsers([...users, proxyUser])
+      .write();
+
+    const adminController = ControllerFactory.admin(conn);
+    const response = await adminController.getAllEmails(proxyUser);
+    expect([...emails, proxyUser.email].sort()).toEqual(response.emails.sort());
+  });
+
+  test('no other emails present asides from registered users', async () => {
```

Comment:
I don't understand the point of this test. If the email doesn't exist, it wouldn't be returned because it doesn't exist. I'd understand if the endpoint returned some filtered list of addresses but that's not the case.

Reply:
Agreed, no filtering is happening or is planned, so I don't think this test is necessary

Reply:
Removed in 3a27e7f.